dataland-qa-service/src/main/kotlin/org/dataland/datalandqaservice/api/QaReportsMetadataApi.kt

```diff
@@ -0,0 +1,58 @@
+package org.dataland.datalandqaservice.org.dataland.datalandqaservice.api
+
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import io.swagger.v3.oas.annotations.security.SecurityRequirement
+import org.dataland.datalandbackend.openApiClient.model.QaStatus
+import org.dataland.datalandqaservice.org.dataland.datalandqaservice.model.DataAndQaReportMetadata
+import org.springframework.format.annotation.DateTimeFormat
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestParam
+import java.util.*
+
+/**
+ * Defines the restful dataland QAReports Metadata API
+ */
+@Suppress("kotlin:S6517")
+@SecurityRequirement(name = "default-bearer-auth")
+@SecurityRequirement(name = "default-oauth")
+interface QaReportsMetadataApi {
+    /**
+     * A method to retrieve metadata of QA reports based on filters.
+     * @param uploaderUserIds If set, only metadata of reports are returned that relate to a dataset of an uploader
+     *  in uploaderUserIds.
+     * @param showOnlyActive If true, only metadata of reports are returned that are active
+     * @param qaStatus If set, only metadata of reports are returned that relate to a dataset with this qaStatus
+     * @param startDate If set, only metadata of reports are returned that are past the startDate
```

Comment:
to be more specific: .. that were uploaded after the startDate

Reply:
Changed.